### PR TITLE
Adding new official WALLDEX API nodes

### PR DESCRIPTION
**Pull request by Eliot-WALLDEX**
### Current Operational Status - https://status.walldex.pro

Wallet - https://w.walldex.pro
Discord - https://discord.gg/cf5s5Rw
Telegram - https://t.me/walldex

### CODE REVIEW

**Comment by sschiessl-bcp** [comment]
Hello dear WallDEX team, 

and welcome to the BitShares blockchain.

Did you have an introductory post somewhere that I might have missed?

**Comment by Eliot-WALLDEX** [comment]
Stefan, yes, first will be on the BitsharesTalk. Awaiting registration access in next few days.

**Comment by sschiessl-bcp** [comment]
Any updates?

**Comment by Eliot-WALLDEX** [comment]
> 
> 
> Any updates?

Yep, tomorrow (June 11). Thank you

**Comment by sschiessl-bcp** [comment]
> > Any updates?
> 
> Yep, tomorrow (June 11). Thank you

Could you point me to it please? If not done that is fine too, not trying to push just collecting information

**Comment by Eliot-WALLDEX** [comment]
> 
> 
> > > Any updates?
> > 
> > 
> > Yep, tomorrow (June 11). Thank you
> 
> Could you point me to it please? If not done that is fine too, not trying to push just collecting information

Everything will be fine. We are announcing. In touch this week.

**Comment by Eliot-WALLDEX** [comment]
Done. @sschiessl-bcp 
https://bitsharestalk.org/index.php?topic=32448.0

File: app/api/apiConfig.js [hunk 1]
```diff
@@ -128,6 +128,317 @@ export const settingsAPIs = {
...
+            url: "wss://turkey.walldex.pro/ws",
+            region: "Western Europe",
+            country: "Turkey",
+            location: "Istanbul",
+            operator: "Witness: walldex",
+            contact: "email:nodes@walldex.pro;telegram:walldex"
+        },
+        {
+            url: "wss://israel.walldex.pro/ws",
+            region: "Western Europe",
```
Comment (sschiessl-bcp) [inline app/api/apiConfig.js:436]: Can you please double-check all regions according to above list `nodeRegions`. The link there shows the classification (for easy "local" geolocation)

**Comment by Eliot-WALLDEX** [comment]
Fixed

**Comment by abitmore** [comment]
https://w.walldex.pro/ is unreachable.

**Comment by sschiessl-bcp** [comment]
All of the URLs related are unreachable. Closing this.

**Pull request closed without merging**